lib/cubits/database_connection/database_connection_cubit.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:bloc/bloc.dart';
+import 'package:coffeecard/base/strings.dart';
+import 'package:coffeecard/data/repositories/v1/app_config_repository.dart';
+import 'package:coffeecard/generated/api/coffeecard_api_v2.swagger.swagger.dart';
+import 'package:coffeecard/service_locator.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+
+part 'database_connection_state.dart';
+
+class DatabaseConnectionCubit extends Cubit<DatabaseConnectionState> {
+  final AppConfigRepository _configRepository = sl.get<AppConfigRepository>();
+
+  DatabaseConnectionCubit()
+      : super(
+          const DatabaseConnectionState(
+            status: DatabaseConnectionStatus.unknown,
+          ),
+        ) {
+    getConfig();
+  }
+
+  Future<void> getConfig() async {
+    final AppConfig config = await _configRepository.getAppConfig();
+
+    final DatabaseConnectionStatus isConnectedToTestDB =
+        config.environmentType == 'Production'
```

Comment:
It is sadly a bug with our swagger generation library, that this is not correctly parsed to the enum type 😢

Reply:
any suggestions what we can do instead or would this be fine until it is resolved? added a fixme until we can resolve it.

Reply:
What you are doing now, is just fine. We'll clean up when the bug has been fixed.